docs: labs reorg (#745)
redpanda-data/documentation: docs/labs/install-preview.mdx

---
title: Install Labs
---

<head>
    <meta name="title" content="Installing Labs | Redpanda Docs"/>

>     <meta name="description" content="We're constantly updating and improving Redpanda with new features, and capabilities."/>

Reviewer: ```suggestion
    <meta name="description" content="Redpanda is constantly updating and adding new features and capabilities."/>
```

</head>

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

:::caution

> Lab features may be tied to specific version releases of the product and are not supported in production deployments.

Reviewer: ```suggestion
Lab features may be tied to specific product release versions and are not supported in production deployments.
```

:::

## What is a Redpanda lab?

> Redpanda continuously adds new features and feature updates. Our lab features are designed for user feedback in development and testing environments. For example, [Tiered Storage](../../data-management/tiered-storage) was a lab in preview before it was promoted to general availability. Your feedback helps shape the future of the Redpanda platform. 

Reviewer: ```suggestion
Redpanda continuously adds new features and feature updates. Lab features are designed for user testing and feedback in development and testing environments. For example, [Tiered Storage](../../data-management/tiered-storage) was a lab feature in preview before it was promoted to general availability. Your feedback helps shape the future of the Redpanda platform. 
```

Lab features are subject to change and may not become available in future releases.

> Send us your feedback on our [Slack Community](https://join.slack.com/t/redpandacommunity/shared_invite/zt-ng2ze1uv-l5VMWSGQHB9gp47~kNnYGA/) or [GitHub Discussions](https://github.com/vectorizedio/redpanda/discussions). 

Reviewer: ```suggestion
Send your feedback on our [Slack Community](https://join.slack.com/t/redpandacommunity/shared_invite/zt-ng2ze1uv-l5VMWSGQHB9gp47~kNnYGA/) or [GitHub Discussions](https://github.com/vectorizedio/redpanda/discussions). 
```
